docs/plans/2026-02-16-coderabbit-defensive-fixes.md

```
@@ -0,0 +1,222 @@
+  }
+}
+```
+
+**Step 3: flutter analyze 확인**
+
+Run: `flutter analyze`
+Expected: No issues
+
+**Step 4: 커밋**
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)